playbooks/roles/memcached/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: install memcached
+  apt: pkg=memcached state=present
```

Comment:
We might want to rewrite this file to follow edX's style guide for Ansible: 
https://openedx.atlassian.net/wiki/display/OpenOPS/Ansible+Code+Conventions

I just became aware of the doc at the conference, so I haven't had a lot of time to read through it.

I remember getting the following feedback on my Ansible code:

1. Use the YAML syntax for tasks. e.g.:
```
apt: pkg=memcached state=present
``` 
would turn into something like: 
```
apt: 
  pkg: memcached 
  state: present
```
2. Don't use handlers. Just restart the service as the last task. Optionally, set/check tags to only do the reset after some config change: 
https://openedx.atlassian.net/wiki/display/OpenOPS/Role+Life-cycle